### ISSUE TITLE
Set dropout parameters

### ISSUE THREAD

**Issue by zhangxinqila** [lucidrains/reformer-pytorch#1]
hi!
the default value of dropout in class init method of LSHAttention is 0 and then no where to change it 
```python
class LSHSelfAttention(nn.Module):
    def __init__(self, emb, heads = 8, bucket_size = 64, n_hashes = 8, causal = False, **kwargs):
        # init position
        self.lsh_attn = LSHAttention(bucket_size=bucket_size, causal=causal, **kwargs)

class Reformer(nn.Module):
    def __init__(self, emb, depth, max_seq_len, num_tokens = 10000, heads = 8, bucket_size = 64, n_hashes = 8, ff_chunks = 100, causal = False, weight_tie = False):
         # Never pass the dropout parameters so the dropout can't be changed
        get_attn = lambda: LSHSelfAttention(emb, heads, bucket_size, n_hashes, causal = causal)

```
Thanks!

**Comment by lucidrains** [comment]
ok fixed!
